aepsych/server/server.py

```diff
@@ -329,24 +332,50 @@ def versioned_handler(self, request):
 
     def handle_setup_v01(self, request):
         logger.debug("got setup message!")
-
+        ### make a temporary config object to derive parameters because server handles config after table
+        tempconfig = Config(**request["message"])
```

Comment:
Will this break if there's no `"config_str"` or `"config_dict"` in the message (we do checks for this in line 373)?

Reply:
@crasanders may I get some clarification on this front?